tests/ha/single_pod_restart/test_server_pod_restart_api.py

```diff
@@ -1130,5 +1259,192 @@ def test_obj_ver_suspension_server_pod_restart(self):
         """
         LOGGER.info("STARTED: Test to verify bucket versioning suspension before & after server "
                     "pod restart.")
+        event = threading.Event()
+        LOGGER.info("Step 1: Create bucket and upload object %s of %s size. Enable versioning "
+                    "on %s.", self.object_name, self.f_size, self.bucket_name)
+        self.extra_files.append(self.multipart_obj_path)
+        args = {'chk_null_version': True, 'is_unversioned': True, 'file_path':
+            self.multipart_obj_path, 'enable_ver': True, 's3_ver': self.s3_ver}
+        resp = self.ha_obj.create_bkt_put_object(event, self.s3_test_obj, self.bucket_name,
+                                                 self.object_name, **args)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 1: Created bucket and uploaded object %s of %s size. Enabled "
+                    "versioning on %s.", self.object_name, self.f_size, self.bucket_name)
+        self.version_etag.update({self.bucket_name: []})
+        self.version_etag[self.bucket_name].extend(resp[1])
+        self.version_etag.update({"obj_name": self.object_name})
+        self.version_etag.update({"s3_ver": self.s3_ver})
+        self.is_ver = True
+        bucket_list = list()
```

Comment:
can we have this list defined in setup method?

Reply:
> can we have this list defined in setup method?

This is not required @sonalk0209. There was one check to verify version and etag from self.version_etag,
since it has other keys too, I added this local variable. It won't be in use for other test except 2 versioning tests